### PR TITLE
Feature/tests and stuff

### PR DESCRIPTION
**Pull request by shrikeh**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,11 @@
         "php": ">=5.3.3"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^2.3"
+        "squizlabs/php_codesniffer": "^2.3",
+        "fabpot/php-cs-fixer": "^1.11"
```
Comment (localheinz) [inline composer.json]: @shrikeh 

How about using the `--sort-packages` option?

```
$ composer require --dev --sort-packages fabpot/php-cs-fixer:^1.11
```

File: .travis.yml [hunk 2]
```diff
@@ -23,5 +33,5 @@ install:
   - composer install --prefer-dist
 
 script:
-  - vendor/bin/phpcs --standard=PSR2 src -p
+  - bin/phpcs --standard=phpcs.xml src
```
Comment (localheinz) [inline .travis.yml]: @shrikeh 

Question is, do we want to use both `phpcs` and `php-cs-fixer` or will one tool be enough?

File: composer.json [hunk 3]
```diff
@@ -12,10 +12,17 @@
...
-        "php": ">=5.3.3"
+        "php": ">=5.3"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^2.3"
+        "squizlabs/php_codesniffer": "^2.3",
+        "escapestudios/symfony2-coding-standard": "~2.0"
+    },
+    "suggest": {
+        "fabpot/php-cs-fixer": "^1.11 - For easy CS fixing"
```
Comment (localheinz) [inline composer.json]: @shrikeh 

You could have a look at https://github.com/refinery29/php-cs-fixer-config to see how we're making use of `fabpot/php-cs-fixer`, maybe worth adding a `Makefile`, too?

Comment (localheinz) [inline composer.json]: @shrikeh 

Alternatively, `squizlabs/php_codesniffer` ships with `phpcbf`, see https://github.com/squizlabs/PHP_CodeSniffer/wiki/Fixing-Errors-Automatically#using-the-php-code-beautifier-and-fixer.

Comment (shrikeh) [inline composer.json]: I shall take a looksee!

File: composer.json (hunk 1 above)
Comment (shrikeh) [inline composer.json]: Not familiar with that one?

**Pull request merged**